litex_boards/targets/qmtech_rp2040_daughterboard.py

```diff
@@ -0,0 +1,136 @@
+                Subsignal(      "gpio0",  Pins("J3:7")),
+                Subsignal(      "gpio1",  Pins("J3:8")),
+                Subsignal(      "gpio2", Pins("J2:44")),
+                Subsignal(      "gpio3", Pins("J2:43")),
+                Subsignal(      "gpio4", Pins("J2:42")),
+                Subsignal(      "gpio5", Pins("J2:41")),
+                Subsignal(      "gpio6", Pins("J2:40")),
+                Subsignal(      "gpio7", Pins("J2:39")),
+                Subsignal(      "gpio8", Pins("J2:38")),
+                Subsignal(      "gpio9", Pins("J2:37")),
+                Subsignal(     "gpio10", Pins("J2:36")),
+                Subsignal(     "gpio11", Pins("J2:35")),
+                Subsignal(     "gpio12", Pins("J2:34")),
+                Subsignal(     "gpio13", Pins("J2:33")),
+                Subsignal(     "gpio14", Pins("J2:32")),
+                Subsignal(     "gpio15", Pins("J2:31")),
+                # RP2040 GPIOs 16 to 19 are external headers
+                # They can be used to drive JTAG interface using Dirty Jtag (for RP2040)
+                # More information at: https://github.com/phdussud/pico-dirtyJtag
+                # A second option is to use the same pins to drive a
+                # Xilinx Virtual Cable (XVC) client for JTAG Programming in Xilinx FPGAs Core Boards
+                # More information at: https://github.com/kholia/xvc-pico
+                # Subsignal(   "gpio16", Pins("Jx:x")),
+                # Subsignal(   "gpio17", Pins("Jx:x")),
+                # Subsignal(   "gpio18", Pins("Jx:x")),
+                # Subsignal(   "gpio19", Pins("Jx:x")),
+                Subsignal(     "gpio20", Pins("J2:30")),
+                Subsignal(     "gpio21", Pins("J2:29")),
+                Subsignal(     "gpio22", Pins("J2:28")),
+                Subsignal(     "gpio23", Pins("J2:27")),
+                # RP2040 GPIOs 24 and 25 are connected to User Push Button and User LED
+                # Subsignal(   "gpio24", Pins("Jx:x")),
+                # Subsignal(   "gpio25", Pins("Jx:x")),
+                Subsignal("gpio26_adc0", Pins("J2:26")),
+                Subsignal("gpio27_adc1", Pins("J2:25")),
+                Subsignal("gpio28_adc2", Pins("J2:16")),
+                Subsignal("gpio29_adc3", Pins("J2:15")),
+                io_standard,
+            ),
+        ]
+
+    connectors = [
+        ("pmoda", "J2:17 J2:19 J2:21 J2:23 J2:18 J2:20 J2:22 J2:24"), #J10
+        ("pmodb", "J2:7  J2:9  J2:11 J2:13 J2:8  J2:10 J2:12 J2:14"), #J11
+        ("J1", {
+             3: "J2:60",
+             4: "J2:59",
+             5: "J2:58",
+             6: "J2:57",
+             7: "J2:56",
+             8: "J2:55",
+             9: "J2:54",
+            10: "J2:53",
+            11: "J2:52",
+            12: "J2:51",
+            13: "J2:50",
+            14: "J2:49",
+            15: "J2:48",
+            16: "J2:47",
+            17: "J2:46",
+            18: "J2:45"
+        }),
+    ]
```

Comment:
Often, the list of I/Os and connectors go to a separate "platform/...py" source file.

Is it intended to keep it here?

Reply:
I did not know, I will drop another commit

Reply:
Maybe moving `litex_boards/targets/qmtech_rp2040_daughterboard.py` to `litex_boards/platforms/qmtech_rp2040_daughterboard.py` is enough? As there is not really possible to run this script and get a board out of it.

Reply:
yes, I am doing that!